logstash/event.py

```diff
@@ -26,7 +30,9 @@ def is_cancelled(self):
         return self.cancelled
 
     def __str__(self):
-        return self.data["@timestamp"] + "," + self.data["@source"] + "," + self.data["@message"]
+        return ",".join([self.data["@timestamp"],
+                         self.data["@source"],
+                         self.data["@message"]])
```

Comment:
This is a scary old format.  Logstash hasn't used `@source` or `@message` since Logstash v1.1 (now on 2.3.4).  The only `@` fields now are `@timestamp` and `@version`, with `@version` indicating this change, and identifying it in the document.